backup/backup.go

```diff
@@ -13,32 +12,21 @@ import (
 
 type Backup interface {
 	Backup() error
-	Restore() error
+	Restore(key []byte) error
```

Comment:
comment explaining what key we are talking about here? is it the pub, priv, or aes key?